build_library/sysext_mangle_kmod

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+configure_modprobe() {
+  local sysext_name="${1}"
+  shift
+
+  local module_directories=(./usr/lib/modules/*-flatcar/)
+
+  mkdir -p ./usr/lib/modprobe.d/
+  for module_name in $(find "${module_directories[@]}" -type f \( -name "*.ko" -o -name "*.ko.*" \) -printf "%f\n" | sed -E 's/\.ko(\.\w+)?$//'); do
+    cat <<EOF >> "./usr/lib/modprobe.d/10-${sysext_name}-kmod-sysext.conf"
+install $module_name     /usr/local/bin/_${sysext_name}_modprobe_helper $module_name
+remove  $module_name     /usr/local/bin/_${sysext_name}_modprobe_helper -r $module_name
+EOF
+  done
+
+  mkdir -p ./usr/local/bin/
```

Comment:
I just realized /use/libexec would be a better location for these internal helpers.

There are scenarios where /ust/local needs to be writeable and that is accomplished by using a sysext that redirects /usr/local to a different path using a symlink. Using the same path would conflict with those scenarios.